src/main/java/ru/lanwen/verbalregex/VerbalExpression.java

```diff
@@ -13,10 +13,10 @@
 
     public static class Builder {
 
-        private StringBuilder prefixes = new StringBuilder();
-        private StringBuilder source = new StringBuilder();
-        private StringBuilder suffixes = new StringBuilder();
-        private int modifiers = Pattern.MULTILINE;
+        private StringBuilder prefixes;
+        private StringBuilder source;
+        private StringBuilder suffixes;
+        private int modifiers;
```

Comment:
The test coverage decreased because the operation "CREATE STRINGBUILDER" has been removed.